plugins/kubernetes/app/models/kubernetes/role_validator.rb

```diff
@@ -398,6 +417,17 @@ def validate_host_volume_paths
 
     # helpers below
 
+    def pod_template(template)
```

Comment:
that's what `def templates` already does

Reply:
i don't understand how...

Reply:
delegates to role_config_file.rb self.templates ... which does some magic to do it ... there are tests, so pretty sure it works :)

Reply:
updated.